app/graphql/resolvers/concerns/collection.rb

```diff
@@ -7,13 +7,15 @@ module Collection
       extend ActiveSupport::Concern
 
       include Sorting
+      include TagFiltering
 
       included do
         type ["Types::#{self::MODEL_CLASS}".safe_constantize], null: false
         argument :search, String, 'Search query', required: false
         argument :limit, Integer, 'Pagination limit', required: false
         argument :offset, Integer, 'Pagination offset', required: false
         argument :sort_by, [String], 'Sort results', required: false
+        argument :tags, [String], 'Filter by tags', required: false
```

Comment:
Can `tags` be only available on types that supports it?

Reply:
Okay, how can we do that? :sweat_smile:  :see_no_evil:

Reply:
Can you try moving it to a concrete type? Would that work?

Reply:
That's was the first I tried :confused:

Reply:
`tags` is available in REST for all endpoints, even though only systems supports/does anything with it. This seems to be the same - why does it matter to remove it here but not in REST?

Reply:
If `tags` param in REST is available on all resources, then the difference would that it is checked (I believe) there to use the tags scoping only for those that actually support it (have the field available).

Reply:
It IS checked here as well, we're kinda using the same resolver. If the `tags` field is not available on the DB model, we're ignoring the passed tags ...

Reply:
Nope, missed that :facepalm: so this opens a window for the question: "how it should be?" :laughing:

Reply:
The quickest way how to achieve for the argument to be only available the type that supports it would be to add a condition here similar to the `tags_supported?` (by checking the field availability on the db model).

Reply:
Fixed...

Reply:
Is it possible to also add a similar condition to the `argument` call?
```
argument ... if self::MODEL_CLASS.column_names.include?('tags')
```